libobs/obs-output.c

```diff
@@ -1042,42 +1048,18 @@ void obs_output_set_audio_conversion(
 	output->audio_conversion_set = true;
 }
 
-static inline size_t num_audio_mixes(const struct obs_output *output)
-{
-	size_t mix_count = 1;
-
-	if ((output->info.flags & OBS_OUTPUT_MULTI_TRACK) != 0) {
-		mix_count = 0;
-
-		for (size_t i = 0; i < MAX_AUDIO_MIXES; i++) {
-			if (!output->audio_encoders[i])
-				break;
-
-			mix_count++;
-		}
-	}
-
-	return mix_count;
-}
-
 static inline bool audio_valid(const struct obs_output *output, bool encoded)
 {
 	if (encoded) {
-		size_t mix_count = num_audio_mixes(output);
-		if (!mix_count)
-			return false;
-
-		for (size_t i = 0; i < mix_count; i++) {
-			if (!output->audio_encoders[i]) {
-				return false;
+		for (size_t i = 0; i < MAX_OUTPUT_AUDIO_ENCODERS; i++) {
+			if (output->audio_encoders[i]) {
+				return true;
 			}
 		}
+		return false;
 	} else {
```

Comment:
This doesn't even need to be an else, it can just return since the `if` statement will always return.

Reply:
Fixed in 537ed6df1a161ee9322a6106bcc83385fc20a5d5